__tests__/app.test.js

```diff
@@ -152,4 +153,58 @@ describe("app", () => {
         });
     });
   });
+  describe("POST /api/articles/:article_id/comments", () => {
+    test("POST 201: inserts a new comment to the db and returns the new comment back to the client", () => {
+      const newComment = {
+        username: "butter_bridge",
+        body: "Love this article!!!",
+      };
+      return request(app)
+        .post("/api/articles/2/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.comments.author).toBe("butter_bridge");
+          expect(body.comments.body).toBe("Love this article!!!");
+        });
+    });
+    test("POST 400: returns bad request if any required field in the comments body is empty", () => {
+      const newComment = {
+        username: "butter_bridge",
+      };
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad Request");
+        });
+    });
+    test("GET 404: returns Not Found when given a non-existent article_id", () => {
+      const newComment = {
+        username: "butter_bridge",
+        body: "Love this article!!!"
+      };
+      return request(app)
+        .post("/api/articles/999/comments")
+        .send(newComment)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Invalid article_id");
```

Comment:
I would adjust the language of the error message away from "invalid", as a `404` denotes data that cannot be found as opposed to an invalid data entry